docs/contribute/01_environment.qmd

```diff
@@ -101,16 +101,24 @@ for manager, params in managers.items():
     print()
     print("    ```sh")
     print("    cd ibis")
-    print(f"    {manager} env create --file environment.yml")
+    print(f"    {manager} env create --file conda/environment.yml")
     print("    ```")
     print()
+    if manager == "conda":
+        print()
+        print(
+            "    If you're using arm64 architecture (Mac M1/M2), use `conda/environment-arm64.yml` for setting up a dev environment for all the backends that are possible to install excluding Flink; use `conda/environment-arm64-flink.yml` for setting up a dev environment for all the backends that are possible to install including Flink. The reason to have two separate environments is because `apache-flink` forces `pyarrow` to downgrade to 11.0, which causes conflicts in other backends."
+        )
```

Comment:
Can you wrap this string so it isn't on a single line?  
Either separate print statements or implicitly joined strings should work.